agent/lib/kontena/workers/service_pod_worker.rb

```diff
@@ -273,5 +334,12 @@ def sync_state_to_master(current_state, error = nil)
     def log_service_pod_event(type, data, severity = Logger::INFO)
       super(service_pod.service_id, service_pod.instance_number, type, data, severity)
     end
+
+    private
+
+    def shutdown
+      @restart_backoff_timer.cancel if @restart_backoff_timer
```

Comment:
I doubt cleaning up the timers in the actor shutdown is necessary... looking at the code, the `@timers` are ignored once the `Celluloid::Actor#run` loop calls `shutdown`?

```
Celluloid::Cell#shutdown Kontena::Workers::ServicePodWorker @ 
	/usr/lib/ruby/gems/2.4.0/gems/celluloid-0.17.3/lib/celluloid/actor.rb:306:in `shutdown'
	/usr/lib/ruby/gems/2.4.0/gems/celluloid-0.17.3/lib/celluloid/actor.rb:169:in `run'
	/usr/lib/ruby/gems/2.4.0/gems/celluloid-0.17.3/lib/celluloid/actor.rb:131:in `block in start'
	/usr/lib/ruby/gems/2.4.0/gems/celluloid-essentials-0.20.5/lib/celluloid/internals/thread_handle.rb:14:in `block in initialize'
	/usr/lib/ruby/gems/2.4.0/gems/celluloid-0.17.3/lib/celluloid/actor/system.rb:78:in `block in get_thread'
	/usr/lib/ruby/gems/2.4.0/gems/celluloid-0.17.3/lib/celluloid/group/spawner.rb:50:in `block in instantiate'
```